src/functions.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * @copyright 2015-2017 Contentful GmbH
+ * @license   MIT
+ */
+
+namespace Contentful;
+
+/**
+  * Unfortunately PHP has no easy way to create a nice, ISO 8601 formatted date string with milliseconds and Z
+  * as the time zone specifier. Thus this hack.
+  *
+  * @param \DateTimeImmutable $date
+  *
+  * @return string ISO 8601 formatted date
+  */
+ function format_date_for_json(\DateTimeImmutable $date)
```

Comment:
About the inconsistency in naming, that's a general convention in PHP, using `snake_case` for simple functions and `camelCase` when they are class methods 🙂 

About the reasons of the change, I'm generally against having static helper classes, mainly because in my experience that never ends well. Overall I try to have classes represent _something_, rather than being ways to group together functions: that's what namespaces are for. Unfortunately supporting PHP 5.5 means I can't add nice `use function Contentful\format_date_for_json;` statements at the beginning like I'm doing [here](https://github.com/contentful/contentful-management.php/pull/47/commits/cc1ac0233a30640266be50a443fd6baf2686d8d0#diff-f8505abedeb59d0333424cdf94cdeabeR12), but that's the nature of the beast.

As you can see, I'm using `GuzzleHttp\json_encode` and `GuzzleHttp\json_decode`, which follow the same logic. Overall it's probably a matter of preferences, though I can't shake the feeling that if I'm creating _a class around a static function (and this function does not access anything inside that class)_, then this function should be better off as standalone 🤔

At least this is for PHP, where OO and procedural code coexist _peacefully_ 😆